skyportal/handlers/api/annotation.py

```diff
@@ -115,6 +114,12 @@ def post(self):
             return self.error("Input `origin` must begin with alphanumeric/underscore")
 
         annotation_data = data.get("data")
+
+        if type(annotation_data) is not dict:
```

Comment:
```suggestion
        if isinstance(annotation_data, dict):
```
is more idiomatic, but this is fine too